packages/scheduler/package.json

```diff
@@ -28,6 +28,7 @@
     "tracing.js",
     "tracing-profiling.js",
     "unstable_mock.js",
+    "unstable_no_dom.js",
```

Comment:
I considered calling the NoDOM version Native since the primary user will be React Native, what do you think about the naming @acdlite?

Reply:
To me the primary purpose is to act as a last resort if you accidentally call this in a Node.js environment, like if you run a test without wrapping in `act`.

React Native will eventually need its own scheduler, so I'd rather not call this one a React Native scheduler since it's not optimized for that at all.